### ISSUE TITLE
Update web3 to be compatible with web3-flashbots

### ISSUE THREAD

**Issue by WyseNynja** [eth-brownie/brownie#1048]
### Overview

I am wanting to use https://github.com/flashbots/web3-flashbots.git with a brownie project.

However, I get this error when trying to install both:

```
The conflict is caused by:
    eth-brownie 1.14.4 depends on web3==5.11.1
    flashbots 0.3.0 depends on web3<6.0.0 and >=5.17.0
```

### Specification

Can we upgrade the web3 in brownie to >=5.17.0?

Any known blockers on that? I half remember web3 upgrades being difficult.

**Comment by WyseNynja** [comment]
For now, I think I might be able to do `export BROWNIE_LIB=1`

**Comment by WyseNynja** [comment]
`export BROWNIE_LIB=1` is working for me

**Comment by WyseNynja** [comment]
Actually that upgrades too much and breaks `brownie test`. I'm trying now with running `pip-compile --upgrade-package web3` for brownie.

**Comment by iamdefinitelyahuman** [comment]
Fixed by #1064
